src/common/helpers/utils.js

```diff
@@ -19,3 +19,24 @@ const _bbidRegex =
 export function isValidBBID(bbid: string): boolean {
 	return _bbidRegex.test(bbid);
 }
+
+export function getEntityModels(orm: Object): Object {
+	const {Author, Edition, EditionGroup, Publisher, Work} = orm;
+	return {
+		Author,
+		Edition,
+		EditionGroup,
+		Publisher,
+		Work
+	};
+}
+
+export function getEntityModelByType(orm: Object, type: string): Object {
+	const entityModels = getEntityModels(orm);
+
+	if (!entityModels[type]) {
+		throw new Error(`Unrecognized entity type: '${type}'`);
+	}
+
+	return entityModels[type];
+}
```

Comment:
There are now two versions of these utility functions
The ones in  /src/server/helpers/utils.js have JSDoc comments.

It would be good to copy over the JSDoc in src/common/helpers/utils.js, and use only those everywhere in src/server (that's why they're in /common, so they can be used by API and server)